installer/bundle/bundle_skel.sh

```diff
@@ -885,6 +917,21 @@ case "$installMode" in
                 [ $TEMP_STATUS -ne 0 ] && BUNDLE_EXIT_STATUS="$TEMP_STATUS"
             fi
         done
+        for i in bundles/*-bundle-test.sh; do
+            # If filespec didn't expand, break out of loop
+            [ ! -f $i ] && break
+
+            # It's possible we have a test file without bundle; if so, ignore it
+            BUNDLE=`basename $i -bundle-test.sh`
+            [ ! -f bundles/${BUNDLE}-*universal.*.sh ] && continue
+
+            ./$i
+            if [ $? -eq 0 ]; then
+                ./bundles/${BUNDLE}-*universal.*.sh --upgrade $FORCE $restartDependencies
+                TEMP_STATUS=$?
+                [ $TEMP_STATUS -ne 0 ] && BUNDLE_EXIT_STATUS="$TEMP_STATUS"
+            fi
```

Comment:
No. The "BUNDLE_EXIT_STATUS" value is handled later. And, the bundle itself will report failure to the console.